src/picomsg/src/oper_db_writer.cpp

```diff
@@ -59,3 +59,50 @@ bool oper_db_writer_terminator_state_change(const uint32_t terminator_id, const
 
 	return true;
 }
+
+bool oper_db_writer_set_terminator_tunnel_counters(const uint32_t terminator_id, const uint64_t count) {
+	auto oper_db_client = picomsg_get_oper_db_client();
+	std::string terminator_id_str = std::to_string(terminator_id);
+	const char* keys[] = { terminator_id_str.c_str()};
+	auto db_obj = oper_db_client_create_db_object(*oper_db_client, LIBOPER_DB_CLIENT_CLASS_ID_TERMINATOR, keys);
+	if (! db_obj) {
+		PICOMSG_LOG(PICOMSG_LOG_ERR, "failed to create oper DB object for class ID LIBOPER_DB_CLIENT_CLASS_ID_TERMINATOR");
+		return false;
+	}
+	
+	if (! db_obj->u64_set(db_obj.get(), LIBOPER_DB_CLIENT_FIELD_ID_TERMINATOR_TUNNEL_COUNTER, count)) {
+		PICOMSG_LOG(PICOMSG_LOG_ERR, "failed to set oper DB object LIBOPER_DB_CLIENT_FIELD_ID_TERMINATOR_TUNNEL_COUNTER to %lu", count);
```

Comment:
Suggest to move the variables passed to the log functions (here and in all the other places) to a new line, as these line are getting way too long